### ISSUE TITLE
오토 식별화 프로그램에 대한 질문

### ISSUE THREAD

**Issue by JB0527** [ni-inha/Automatic-identification-system-for-fall-event-records-in-EMR-clinical-notes#3]
여기서 bert가 활용되는데, 일단 이 분류 부분에서 KLUE BERT가 쓰인 이유와  이 코드 부분을 정확히 이해 못했습니다! API를 가져오는 법이나 허깅페이스에 대한 스스로의 공부도 필요할 것 같습니다.

'''
class TFBertForSequenceClassification(tf.keras.Model):
    def __init__(self, HUGGINGFACE_MODEL_PATH):
        super(TFBertForSequenceClassification, self).__init__()
        self.bert = TFBertModel.from_pretrained(
            HUGGINGFACE_MODEL_PATH, from_pt=True)
        self.classifier = tf.keras.layers.Dense(1,
                                                kernel_initializer=tf.keras.initializers.TruncatedNormal(
                                                    0.02),
                                                activation='sigmoid',
                                                name='classifier')

    def call(self, inputs):
        input_ids, attention_mask, token_type_ids = inputs
        outputs = self.bert(
            input_ids=input_ids, attention_mask=attention_mask, token_type_ids=token_type_ids)
        cls_token = outputs[1]
        prediction = self.classifier(cls_token)

        return prediction
'''
추가로,  데이터베이스 부분도 쿼리문을 아직 정확히 배우진 않아서 제대로 코드를 이해하지 못했습니다.
database/nlp_fallresult.sql 이 부분에 대해 학습이 필요할 것 같고 대부분 이렇게 데이터베이스 쿼리문으로 빼오는 지도 궁금합니다!

또, 아까 익명화프로그램에서  Free_col을 불러온 'nsc_Emr'의 쓰임을 확인했습니다.

감사합니다.
